src/crl/interactivesessions/_version.py

```diff
@@ -1,6 +1,6 @@
 __copyright__ = 'Copyright (C) 2019-2020, Nokia'
 
-VERSION = '1.3.2'
+VERSION = '1.3.3'
```

Comment:
If we publish beta, please indicate it here. 
Moreover, we should consider that whether this is in fact a minor release: we register new shells which means more functionality for users. So, in fact, this should be 1.4.0b1.